types/webextension-polyfill/namespaces/tabs.d.ts

```diff
@@ -1067,16 +1059,17 @@ export namespace Tabs {
          * is sent back.  The $(ref:runtime.onMessage) event is fired in each content script running in the specified tab for the
          * current extension.
          *
-         * @param tabId
-         * @param message
          * @param options Optional.
          */
-        sendMessage(tabId: number, message: any, options?: SendMessageOptionsType): Promise<any>;
+        // eslint-disable-next-line @definitelytyped/no-unnecessary-generics
+        sendMessage<TMessage = unknown, TResponse = unknown>(
```

Comment:
Same as my comment on the other `sendMessage`.

Reply:
same reason as for the other sendMessage